### ISSUE TITLE
Asymmetries occur with thin obstructions

### ISSUE THREAD

**Issue by mcgratta** [firemodels/fds#4549]
Run this case with `CONSTANT_SPECIFIC_HEAT_RATIO=.TRUE.` and it remains symmetric in the x=constant plane. Set to `.FALSE.` and asymmetries arise quickly. I suspect that the transport terms in the divergence are not symmetric when there is a thin obstruction.
```
&HEAD CHID='RSE_15', TITLE='NIST Reduced Scale Enclosure Experiments, 2007' /

&MESH XB= 0.00,1.42,0.00,0.95, 0.00,0.98, IJK = 48,32,32 /

&TIME T_END=4. /

&REAC FUEL='N-HEPTANE' /

&MISC NOISE=.FALSE., STRATIFICATION=.FALSE., CONSTANT_SPECIFIC_HEAT_RATIO=.TRUE. /

&RADI RADIATION=.FALSE. /

&SURF ID='WALL', MATL_ID='GYPSUM', THICKNESS=0.025, COLOR='ANTIQUE WHITE', DEFAULT=.TRUE., FREE_SLIP=.TRUE. /
&SURF ID='STEEL SHEET', MATL_ID='STEEL', THICKNESS=0.00635, COLOR='BLACK', FREE_SLIP=.TRUE. /
&SURF ID='fire', SPEC_ID(1)='N-HEPTANE', MASS_FLUX(1)=0.0082, COLOR='ORANGE', TMP_FRONT=100. /

&OBST XB=0.54,0.89,0.31,0.66,0.00,0.002, SURF_IDS='fire','STEEL SHEET','STEEL SHEET' /
&OBST XB=0.54,0.54,0.31,0.66,0.00,0.12, SURF_ID='STEEL SHEET' /
&OBST XB=0.89,0.89,0.31,0.66,0.00,0.12, SURF_ID='STEEL SHEET' /
&OBST XB=0.54,0.89,0.31,0.31,0.00,0.12, SURF_ID='STEEL SHEET' /
&OBST XB=0.54,0.89,0.66,0.66,0.00,0.12, SURF_ID='STEEL SHEET' /

&VENT XB=0.0, 0.0, 0.24, 0.72, 0.00,0.81, SURF_ID='OPEN' / 

&MATL ID            = 'STEEL'
      SPECIFIC_HEAT = 0.77
      EMISSIVITY    = 0.8
      DENSITY       = 7800.
      CONDUCTIVITY  = 45. /

&MATL ID            = 'GYPSUM'
      SPECIFIC_HEAT = 1.
      EMISSIVITY    = 0.9
      DENSITY       = 700.
      CONDUCTIVITY  = 0.1 /

&SLCF PBY=0.47, QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF PBY=0.47, QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', CELL_CENTERED=.TRUE. /
&SLCF PBX=0.71, QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE. /
&SLCF PBX=0.71, QUANTITY='DIVERGENCE', CELL_CENTERED=.TRUE. /
&SLCF PBX=0.71, QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', CELL_CENTERED=.TRUE. /

&SLCF PBY=0.47, QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., VECTOR=.TRUE. /
&SLCF PBY=0.47, QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', CELL_CENTERED=.TRUE., VECTOR=.TRUE. /
&SLCF PBX=0.71, QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., VECTOR=.TRUE. /
&SLCF PBX=0.71, QUANTITY='DIVERGENCE', CELL_CENTERED=.TRUE., VECTOR=.TRUE. /
&SLCF PBX=0.71, QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', CELL_CENTERED=.TRUE., VECTOR=.TRUE. /

&SLCF PBY=0.47, QUANTITY='TEMPERATURE', VECTOR=.TRUE. /
&SLCF PBY=0.47, QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', VECTOR=.TRUE. /
&SLCF PBX=0.71, QUANTITY='TEMPERATURE', VECTOR=.TRUE. /
&SLCF PBX=0.71, QUANTITY='DIVERGENCE', VECTOR=.TRUE. /
&SLCF PBX=0.71, QUANTITY='VOLUME FRACTION', SPEC_ID='N-HEPTANE', VECTOR=.TRUE. /

&BNDF QUANTITY='WALL TEMPERATURE', CELL_CENTERED=.TRUE. /

&TAIL /
```

**Comment by rmcdermo** [comment]
Hopefully fixed with Pull Request #4550.

**Comment by mcgratta** [comment]
This case is resolved.
